kube/src/client/mod.rs

```diff
@@ -77,103 +73,99 @@ impl Client {
         Self::try_from(client_config)
     }
 
-    async fn send(&self, request: http::Request<Vec<u8>>) -> Result<reqwest::Response> {
+    async fn send(&self, request: Request<Body>) -> Result<Response<Body>> {
         let (parts, body) = request.into_parts();
-        let pandq = parts.uri.path_and_query().expect("valid path+query from kube");
-        let uri_str = finalize_url(&self.cluster_url, &pandq);
         //trace!("Sending request => method = {} uri = {}", parts.method, &uri_str);
 
-        let mut headers = parts.headers;
-        // If we have auth headers set, make sure they are updated and attached to the request
-        if let Some(auth_header) = self.config.get_auth_header().await? {
-            headers.insert(reqwest::header::AUTHORIZATION, auth_header);
-        }
-
+        // REVIEW Is the following necessary? Was originally for reqwest's client builder having separate methods.
```

Comment:
https://github.com/clux/kube-rs/blob/ab2a380b0ff2f952f4bf41f2c4f8609e0bd1eaa7/kube/src/client/mod.rs#L86-L93

Reply:
Oh, hah, that never really served any point with reqwest either. We controlled all the calls to reqwest via Resource anyway.  Feel free to rip that out along with the error.